docs/benchmarks/pulsar.md

```diff
@@ -107,6 +107,8 @@ $ ssh -i ~/.ssh/pulsar_aws ec2-user@$(terraform output client_ssh_host)
 ## Running the benchmarks from the client host
 
 > The instructions immediately below are for running the benchmarks from a single host, but you  can also run the benchmarks in [distributed mode](#distributed-mode) (from multiple clients simultaneously).
+>
+> The benchmark scripts are copied under `/opt/benchmark`. Following commands are issued at working directory `/opt/benchmark`.
```

Comment:
I think we only need one sentence here:

"The benchmark scripts can be run from the `/opt/benchmark` working directory."

That should also be added to the Kafka doc.

Reply:
addressed the comments and also added to kafka doc.